NEWS.md

```diff
@@ -1,4 +1,24 @@
-## 7.0.0 - Unreleased
+## 8.0.0 - Unreleased
+
+## 7.0.0 - Released
+The main focus of this release was to implement API for managing Teams (Units, Assignments, Memberships), schemas updating and performance optimization.
+
+[Full Changelog](https://github.com/folio-org/mod-orders-storage/compare/v6.0.0...v7.0.0)
+
+### Stories
+* [MODORDSTOR-100](https://issues.folio.org/browse/MODORDSTOR-100) - Acquisitions unit names should be unique
+* [MODORDSTOR-98](https://issues.folio.org/browse/MODORDSTOR-98) - Schema updates: filterable boolean properties to have default value
+* [MODORDSTOR-96](https://issues.folio.org/browse/MODORDSTOR-96) - Add acquisitions-unit-assignments.recordId to receiving-history view
+* [MODORDSTOR-94](https://issues.folio.org/browse/MODORDSTOR-94) - Add acquisitions-unit-assignments.recordId to orders views
```

Comment:
I think reference to the story https://issues.folio.org/browse/MODORDSTOR-95 should be added as well